styles/styles.css

```diff
@@ -99,6 +100,13 @@
   --overlay-background-color: #eee;
   --highlight-background-color: #ccc;
   --text-color: var(--c-black);
+  --dark-blue-color: #2E308E;
```

Comment:
why not using the `Netcentric colors` variables from above?

Reply:
Thanks, was left over from a merge